diff --git a/app/phpunit.xml.dist b/app/phpunit.xml.dist
index bd24527..b59e0a0 100644
--- a/app/phpunit.xml.dist
+++ b/app/phpunit.xml.dist
@@ -15,6 +15,7 @@
 
     <testsuites>
         <testsuite name="Project Test Suite">
+            <directory>../app/Tests</directory>
             <directory>../src/*Bundle/Tests</directory>
             <directory>../src/*/*Bundle/Tests</directory>
             <directory>../src/*/Bundle/*Bundle/Tests</directory>
