diff --git a/app/Resources/views/blog/post_show.html.twig b/app/Resources/views/blog/post_show.html.twig
index 6c735f6..1a1388e 100644
--- a/app/Resources/views/blog/post_show.html.twig
+++ b/app/Resources/views/blog/post_show.html.twig
@@ -15,7 +15,7 @@
         See http://symfony.com/doc/current/cookbook/security/remember_me.html#forcing-the-user-to-re-authenticate-before-accessing-certain-resources
         #}
         {% if is_granted('IS_AUTHENTICATED_FULLY') %}
-            {{ render(controller('AppBundle:Blog:commentForm', { 'id': post.id })) }}
+            {{ render(controller('App\Controller\BlogController::commentFormAction', { 'id': post.id })) }}
         {% else %}
             <p>
                 <a class="btn btn-success" href="{{ path('security_login_form') }}">
